### PR TITLE
Bug 1792435:  Add 4.4 channels

### PR DESCRIPTION
**Pull request by rhamilto**
cc: @jwforres, @jhadvig

### CODE REVIEW

File: frontend/public/module/k8s/cluster-settings.ts [hunk 1]
```diff
@@ -30,6 +30,9 @@ export const getAvailableClusterUpdates = (cv: ClusterVersionKind): ClusterUpdat
 export const getAvailableClusterChannels = () => ({
   'stable-4.3': 'stable-4.3',
   'fast-4.3': 'fast-4.3',
```
Comment (spadgett) [inline frontend/public/module/k8s/cluster-settings.ts]: We should be remove the 4.3 channels as well.

Comment (rhamilto) [reply frontend/public/module/k8s/cluster-settings.ts]: Thanks, @spadgett.  Done.

**Comment by benjaminapetersen** [comment]
/lgtm

Thanks!

**Comment by jwforres** [comment]
Thanks guys!

**Comment by rhamilto** [comment]
/hold

for ci to be fixed

**Pull request merged**